### PR TITLE
Don't fail on CREATE_INDEX for read-only DB

### PR DESCRIPTION
**Pull request by gitcarbs**

### CODE REVIEW

File: guillotina_elasticsearch/commands/vacuum.py [hunk 1]
```diff
@@ -223,8 +223,12 @@ def __init__(self, txn, tm, request, container, last_tid=-2, index_scroll='15m',
         # WHY this way?
         #   - uses less memory rather than getting all keys in both.
         #   - this way should allow us handle VERY large datasets
-        conn = await self.txn.get_connection()
-        await conn.execute(CREATE_INDEX)
+
+        try:
+            conn = await self.txn.get_connection()
+            await conn.execute(CREATE_INDEX)
+        except:
```
Comment (vangheem) [inline guillotina_elasticsearch/commands/vacuum.py]: Can we use `except Exception` instead?

**Review by vangheem — CHANGES_REQUESTED** [review]

File: guillotina_elasticsearch/commands/vacuum.py (hunk 1 above)
Comment (gitcarbs) [reply guillotina_elasticsearch/commands/vacuum.py]: Done

**Review by vangheem — APPROVED** [review]

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
-1.3.29 (unreleased)
```
Comment (vangheem) [inline CHANGELOG.rst]: Sorry, another. This is not released so please leave the `(unreleased)` here.

**Review by vangheem — CHANGES_REQUESTED** [review]

File: VERSION [hunk 3]
```diff
@@ -1 +1 @@
-1.3.29.dev0
+1.3.29
```
Comment (vangheem) [inline VERSION]: Also, please do not change the version. This is done when I cut a release.

**Review by vangheem — CHANGES_REQUESTED** [review]

**Review by vangheem — APPROVED** [review]

**Comment by vangheem** [comment]
released  1.3.29

**Pull request merged**